include/dlaf/lapack_tile.tpp

```diff
@@ -8,6 +8,36 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 
+#include "dlaf/common/assert.h"
+
+template <class T, Device device>
+dlaf::BaseType<T> lange(lapack::Norm norm, const Tile<T, device>& a) noexcept {
+  return lapack::lange(norm, a.size().rows(), a.size().cols(), a.ptr(), a.ld());
+}
+
+template <class T, Device device>
+dlaf::BaseType<T> lantr(lapack::Norm norm, blas::Uplo uplo, blas::Diag diag,
+                        const Tile<T, device>& a) noexcept {
+  switch (uplo) {
+    case blas::Uplo::Lower:
+      DLAF_ASSERT(a.size().rows() >= a.size().cols(), "Not valid ", a.size());
+      break;
+    case blas::Uplo::Upper:
+      DLAF_ASSERT(a.size().rows() <= a.size().cols(), "Not valid ", a.size());
+      break;
```

Comment:
With the new ASSERT mechanism, the tiles will be printable, we can just place them as a parameter, (e.g. `a` instead of `a.size()`). It might make sense to change the message to make it more as a sentence instead of having values in between strings, for example

"Not valid <some_val>" to just "Not valid!"

Although in this case, I wouldn't mind omitting the message altogether as the condition is quite clear.

Reply:
I'll wait that the #231 gets merged 😉

Reply:
This should have been addressed now 😉